curriculum/challenges/english/07-scientific-computing-with-python/learn-encapsulation-by-building-a-projectile-trajectory-calculator/660400f02cf3a76795b82d4e.md

```diff
@@ -0,0 +1,73 @@
+---
+id: 660400f02cf3a76795b82d4e
+title: Step 4
+challengeType: 20
+dashedName: step-4
+---
+
+# --description--
+
+The first thing to set up is a method that calculates the displacement of the projectile, which is the horizontal space traveled from the throw to when it touches the ground.
+
+Create a function `__calculate_displacement`, which takes no arguments, and return the displacement of the projectile.
```

Comment:
"which takes no arguments" can be misleading considering that it should take `self`.

Reply:
when called it doesn't take any arguments...
```suggestion
Create a function `__calculate_displacement`, which has only `self` as argument, and return the displacement of the projectile.
```